lib/components/FilePicker/FileListRow.vue

```diff
@@ -135,6 +140,11 @@ function handleKeyDown(event: KeyboardEvent) {
 		&--selected {
 			background-color: var(--color-background-dark);
 		}
+
+		&--not-pickable {
+			opacity: 0.5;
```

Comment:
I think this is how we show it in most cases (also making colored previews less colorful):

```suggestion
			filter: saturate(0.7);
			opacity: 0.7;
```